### ISSUE TITLE
Suggestion: Managing only a diff part of the GopherJS playground

### ISSUE THREAD

**Issue by hajimehoshi** [ebiten-playground/ebiten-playground.github.io#9]
As GopherJS and its playground is often updated and it'd be hard to follow that, I suggest to manage only the diff part (IIUC they are exposing Go.Compile or other functions as promises) instead of managing a full set of https://github.com/j7b/jsplayground . What do you think?

**Comment by hajimehoshi** [comment]
Hm,  github.com/j7b/jsplayground is much different thing from github.com/gopherjs/gopherjs.github.io/playground.

**Comment by hajimehoshi** [comment]
Let me change the question: Why is github.com/j7b/jsplayground/generate.go needed instead of github.com/gopherjs/gopherjs.github.io/playground/update.sh ?

**Comment by j7b** [comment]
The original keeps the playground js and the pkg build pretty closely coupled which complicates adding additional packages (ebiten in particular). Mine outputs public members as a json file and uses gopherjs get explicitly for additional packages. The intent of jsplayground was to keep all the parts as decoupled as possible but end up with all the parts somebody would need to make a playground when you go get it.

**Comment by hajimehoshi** [comment]
That makes sense very much. Thank you for elaborating.
